packages/patternfly-4/react-integration/cypress/integration/breadcrumb.spec.ts

```diff
@@ -0,0 +1,19 @@
+describe('Breadcrumb Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#breadcrumb-demo-nav-item-link').click();
+    cy.url().should('eq', 'http://localhost:3000/breadcrumb-demo-nav-link') 
+  });
+
+  it('Verify breadcrumb items have link', () => {
+    cy.get('li').children('a');
+  }); 
+
+  it('Verify default section home content', () => {
+    cy.get('.pf-c-breadcrumb__link').first().contains('Section Home'); 
+  });
+
+  it('Verify default section label defaults', () => {
+    cy.get('.pf-c-breadcrumb__link').contains('Section Title'); 
+  }); 
+});
```

Comment:
Can you add a test to click on the components in the breadcrumb and change the url?